querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx

```diff
@@ -143,7 +148,18 @@ export const DataDocLeftSidebar: React.FunctionComponent<IProps> = ({
                 hidden: cells.length === 0,
             })}
         >
-            {contentDOM}
+            {contentState === 'default' ? (
+                <> {contentDOM} </>
+            ) : (
+                <Resizable
+                    defaultSize={{ width: `${sidebarWidth}px` }}
+                    minWidth={sidebarWidth}
+                    enable={enableResizable({ right: true })}
+                    onResize={resizeToCollapseSidebar}
+                >
+                    {contentDOM}
+                </Resizable>
+            )}
```

Comment:
I think we only need either this ` ? :` check or different `sidebarWidth`.  We can keep this.
Try to remove the `HIDDEN_SIDEBAR_WIDTH` and the `useEffect` logic above. Here will be 
```
                <Resizable
                    defaultSize={{ width: `${DEFAULT_SIDEBAR_WIDTH}px` }}
                    minWidth={DEFAULT_SIDEBAR_WIDTH}
                    enable={enableResizable({ right: true })}
                    onResize={resizeToCollapseSidebar}
                >
                    {contentDOM}
                </Resizable>
```